packages/documentation/pages/usage/components/form.vue

```diff
@@ -323,9 +324,9 @@ export default defineComponent({
 				/* eslint-enable no-magic-numbers */
 			}),
 			genderOptions: computed((): Kotti.FieldRadioGroup.Props['options'] => [
-				{ label: 'MALE', value: 'male' },
+				{ label: 'MALE', value: 'male', dataTest: 'unique-male-data-test' },
 				{ label: 'FEMALE', value: 'female' },
```

Comment:
Is it intended not to give dataTest to this option?

Reply:
yes so i was basically trying to see that i actually support data tests per option (which is respected here)

and if i don't pass it , i checked in the docs that the rendered data-test was 
`field.dataTest.${optionValue}`